dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -306,9 +309,220 @@ public void findOneTestWrongUUID() throws Exception {
         Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
         Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
 
-        getClient().perform(get("/api/core/items/" + UUID.randomUUID()))
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(get("/api/core/items/" + UUID.randomUUID()))
                    .andExpect(status().isNotFound());
         ;
     }
 
+    @Test
+    public void embargoAnonymousAccessTest() throws Exception{
+        context.turnOffAuthorisationSystem();
```

Comment:
should calls to turnOffAuthorizationSystem() be wrapped in a try... finally to ensure that context is always reset properly?  Or, is the context recreated with each test?